CHANGELOG.md

```diff
@@ -3,75 +3,61 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [0.0.1-alpha.5](https://github.com/chanzuckerberg/lp-design-system/compare/v0.0.1-alpha.4...v0.0.1-alpha.5) (2020-10-19)
-
+## [0.0.1-alpha.6](https://github.com/chanzuckerberg/lp-design-system/compare/v0.0.1-alpha.5...v0.0.1-alpha.6) (2021-01-20)
 
 ### Features
 
-* **tokens:** replace eds token numbers ([59b3552](https://github.com/chanzuckerberg/lp-design-system/commit/59b355274973098315ba0bd531a760c2a0d9499e))
-
+- **typography:** add Typography component ([36f5d5c](https://github.com/chanzuckerberg/lp-design-system/commit/36f5d5c9e87e3515f69391426157361c05d523dd))
```

Comment:
I didn't include any of the build/test changes? but I am admittedly a publishing n00b

also idk why there are so many whitespace changes it just auto-formatted on save

Reply:
I'm good with the whitespace changes 👍 

Also, I think it's reasonable to not include build/test changes. Sometimes I include those kinds of things, but it's only really necessary to include user/developer-facing changes (from the standpoint of someone consuming these packages).